docs/en/img

```diff
@@ -1 +1 @@
-../img/
```

Comment:
What is the change here? The diff doesn't make it clear.

Reply:
It's the `/`. The symlinks (which I made months ago) were previously incorrect.